externs/shaka/player.js

```diff
@@ -1098,6 +1099,12 @@ shaka.extern.CmcdConfiguration;
  *   license.  A network will be required to retrieve a temporary license to
  *   view.
  *   Defaults to <code>true</code>.
+ * @property {number} numberOfParallelDownloads
+ *   Number of parallel downloads. Putting a number other than the default can
+ *   help reduce download time.
```

Comment:
If it defaults to `5` now, this statement about "a number other than the default" is no longer valid.